config/sync/field.field.paragraph.os2loop_documents_step.os2loop_documents_step_text.yml

```diff
@@ -21,6 +21,7 @@ third_party_settings:
       os2loop_post: os2loop_post
       os2loop_question_plain_text: os2loop_question_plain_text
       os2loop_question_rich_text: os2loop_question_rich_text
+      os2loop_documents_rich_text: 0
```

Comment:
We should “Limit allowed text formats” on the content type to just the os2loop_documents_rich_text  text format (edit the field, check “Limit allowed text formats” and select only a single format).